msteams-platform/toolkit/tools-prerequisites.md

```diff
@@ -13,14 +13,13 @@ Before you create your Teams app project, ensure that the prerequisites are in p
 
 * [Install required tools to build your Teams app](#install-required-tools-to-build-your-teams-app)
 * [Prepare Accounts to build your Teams app](#accounts-to-build-your-teams-app)
-* [Sideload permission](tools-prerequisites.md)
```

Comment:
Why did we remove it. Please update it as per the provided link?

Reply:
The suggested link is available in Verify sideloading permission. 

https://learn.microsoft.com/en-us/microsoftteams/platform/toolkit/tools-prerequisites#verify-sideloading-permission